pkg/builtin/manifest.yml

```diff
@@ -1260,6 +1260,11 @@ extensions:
     type: widget
     title: Menu
     description: Menu widgets
+    widgetLayout:
+      defaultLocation:
+        zone: outer
+        section: left
+        area: top
```

Comment:
We should keep backward compatibility for already published scenes. Define menu widget as floating and define new widget: Button.